2. flow_control/Q16.c

```diff
@@ -1 +1,12 @@
 // Write a program to read an integer n and print the multiplication table of n up to 10 using a for loop.
+#include<stdio.h>
+int main(){
+    int n,i;
```

Comment:
don't declare "i" variable here.
study about "block scope in programming languages and how we can make use of it"

Reply:
Should I declare it in the for loop?

Reply:
yes